### DISCUSSION TITLE
Mapping rules visibility to improve re-use and consistency of applying mappings

### DISCUSSION THREAD

**Discussion by gordonmilligan-uod** [Health-Informatics-UoN/carrot#62] [Ideas]
As mappings are non-disclosive, we need a way for uses to see the mappings from different data sets that have previously been applied. This will allow them to see a history of the decisions that have been made and chose whether to also apply the same logic to their mappings. It could allow them to also 'import' these rules to their own project for application to their data which is helpful when similar questionnaire's etc are used.

**Comment by spco** [comment]
Great idea, perhaps linked to #627. Would need to consider the format for sharing, plus implement how to import them, and whether that has to be at time of the upload of the SR or afterwards.

**Comment by AndyRae** [comment]
I've moved this to a discussion - as it needs wider thought, design, and planning.
I have some initial ideas around this which I will share as well.
